site/public/css/colors.css

```diff
@@ -1,46 +1,39 @@
-:root{
-    /*master color*/
+:root {
+    /* master color */
     --submitty-logo-blue: #316498;
-    /*might wanna change blue borders/hover text/sidebar text to submitty blue*/
 
-    /*buttons, primarily but also can be used for notifications/anything
-    that requires this color scheme*/
+    /* might wanna change blue borders/hover text/sidebar text to submitty blue */
+
+    /* buttons, primarily but also can be used for notifications/anything
+    that requires this color scheme */
     --danger-red: #dc3545;
-    --focus-danger-red: #ff5555;
+    --focus-danger-red: #f55;
```

Comment:
Might be useful to run a quick poll to see if people like the short color codes vs long color codes. Personally I like the longer one, if just that's what any hex picker I've used gives me, and this is then a style I would violate consistently.

Reply:
I agree.  Should I add a rule to standardize it to always be the full 6 characters?

Reply:
Yes, please.